src/tests/unit/tests/common/components/selector-input-list.test.tsx

```diff
@@ -250,14 +251,19 @@ describe('SelectorInputListTest', () => {
             onDeleteSelector: null,
             onChangeInspectMode: null,
         };
-        const result = Enzyme.shallow(<TestableSelectorInputList {...props} />);
-        const firstState = (result.instance() as TestableSelectorInputList).state
-            .isTextFieldValueValid;
-        (result.instance() as TestableSelectorInputList).setTextFieldValue(givenSelector);
-        (result.instance() as TestableSelectorInputList).componentDidUpdate(previousProps);
-        const secondState = (result.instance() as TestableSelectorInputList).state
-            .isTextFieldValueValid;
-        expect(secondState).toEqual(!firstState);
+        const { rerender, getByPlaceholderText, container } = render(
+            <TestableSelectorInputList {...props} />,
+        );
+        const button = container.querySelector('.textboxAddSelectorButton');
```

Comment:
Prioritize using `getBy` APIs when we can.

```suggestion
        const { rerender, getByPlaceholderText, getByRole } = render(
            <TestableSelectorInputList {...props} />,
        );
        const button = getByRole('button', { name: 'Add Selector' });
```

Reply:
Did the suggested change